spec/read.spec.js

```diff
@@ -1,253 +1,252 @@
-var nock = require('nock');
+'use strict';
 
-describe('CSV Read component', function () {
-    var csv = require('../lib/read.js');
-    var runTest = require('./testrunner.js').runTest;
-    var moment = require('moment');
-    var Q = require('q');
-    var fs = require("fs");
+const expect = require('chai').expect;
+
+let nock = require('nock');
+
+describe('CSV Read component', function CsvReadComponentTests() {
+    let csv = require('../lib/read.js');
+    let runTest = require('./testrunner.js').runTest;
 
     nock('http://test.env.mock')
         .get('/simple.csv')
-        .replyWithFile(200, __dirname + '/test/simple.csv');
+        .replyWithFile(200, __dirname + '/../test/simple.csv');
 
     nock('http://test.env.mock')
         .get('/dates.csv')
-        .replyWithFile(200, __dirname + '/test/dates.csv');
+        .replyWithFile(200, __dirname + '/../test/dates.csv');
 
     nock('http://test.env.mock')
         .get('/numbers.csv')
-        .replyWithFile(200, __dirname + '/test/numbers.csv');
+        .replyWithFile(200, __dirname + '/../test/numbers.csv');
 
     nock('http://test.env.mock')
         .get('/numbers_us.csv')
-        .replyWithFile(200, __dirname + '/test/numbers_us.csv');
+        .replyWithFile(200, __dirname + '/../test/numbers_us.csv');
 
     nock('http://test.env.mock')
         .get('/numbers_de.csv')
-        .replyWithFile(200, __dirname + '/test/numbers_de.csv');
+        .replyWithFile(200, __dirname + '/../test/numbers_de.csv');
 
     function expectDate(date, year, month, day, minutes, seconds) {
-        expect(date.getUTCFullYear()).toEqual(year);
-        expect(date.getUTCMonth()).toEqual(month);
-        expect(date.getUTCDate()).toEqual(day);
-        expect(date.getUTCMinutes()).toEqual(minutes);
-        expect(date.getUTCSeconds()).toEqual(seconds);
+        expect(date.getUTCFullYear()).to.equal(year);
+        expect(date.getUTCMonth()).to.equal(month);
+        expect(date.getUTCDate()).to.equal(day);
+        expect(date.getUTCMinutes()).to.equal(minutes);
+        expect(date.getUTCSeconds()).to.equal(seconds);
     }
 
 
-    it('empty body', function(done) {
-        runTest(csv.process, {}, {}, function(runner) {
-            expect(runner.data.length).toEqual(0);
-            expect(runner.errors.length).toEqual(1);
-            expect(runner.snapshot).toBeUndefined();
+    it('empty body', function emptyBodyTest(done) {
```

Comment:
It's not clear what's being checked in the test. Better to change description into like `should handle empty body...`